### DISCUSSION TITLE
the backups numbers are not correct

### DISCUSSION THREAD

**Discussion by mysteryzzp** [immich-app/immich#12557] [Q&A]
### The bug

in android app. when the backups is running,the picture numbers is not correct

I used other gallery service(likely google photos) before. Many of my photos are in the cloud. I download all the photos to my phone,and then backup them with immich app. There are about 3000 photos. 

![微信图片_20240911101227](https://github.com/user-attachments/assets/0039bfa2-83a9-49fc-bf0b-cd6beae30d23)

### The OS that Immich Server is running on

Linux orangepi3b 5.10.160-rockchip-rk356x

### Version of Immich Server

v1.114.0

### Version of Immich Mobile App

v1.114.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
-
```

### Your .env content

```Shell
-
```

### Reproduction steps

-

**Comment by mysteryzzp** [comment]
I'm wondered how the immich app recognize 'the same one photo'.Does it use the content-id in MediaStore? 
If the backup process interrupted. Does it check the photo status by server api or local db?

**Comment by alextran1502** [comment]
The difference includes corrupted files (files that cannot be hashed) and duplicated files

**Reply by mysteryzzp** [reply]
the uploading task is running. but the 'remain count' is zero. why ?

**Reply by alextran1502** [reply]
I believe there is a bug about counting the assets.
